docs/developer/01-implementing-google-analytics.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Analytics
+label: Google Analytics
+---
+
+### Implementation
+
+By adding the following to elements&hellip;
+
+1. `data-ga="click"`
+  - attaches the Google Analytics click 'event' to this element.
+2. `data-ga-category="accordion"`
+  - sets the component event category recorded by GA.
+3. `data-ga-action="Toggle all panels"`
+  - provides detail on the action this element triggered
+4. `data-ga-label="Show all"`
+  - provides detail on the action this element triggered
+
```

Comment:
@armstrongb Worth adding an example of the head script devs should use, specifically the setting of 'anonymizeIp', considering its not in the default script.

Reply:
LGTM